mpt/impl/smtp/core/pom.xml

```diff
@@ -152,6 +152,15 @@
                 </dependency>
                 <dependency>
                     <groupId>org.apache.james</groupId>
+                    <artifactId>james-server-data-jmap</artifactId>
+                </dependency>
+                <dependency>
+                    <groupId>org.apache.james</groupId>
+                    <artifactId>james-server-jmap-integration-testing</artifactId>
```

Comment:
Ok but here, you are importing a ton of code and dependencies. You can split the part you want to reuse somewhere else. (or as suggested in another commit, just don't use MPT for that at all)